packages/liveblocks-react-lexical/src/liveblocks-plugin-provider.tsx

```diff
@@ -42,14 +40,21 @@ export type EditorStatus =
   /* The editor state is sync with Liveblocks servers */
   | "synchronized";
 
-function getEditorStatus(
-  provider?: LiveblocksYjsProvider<never, never, never, never, never>
-): EditorStatus {
-  if (provider === undefined) {
-    return "not-loaded";
-  }
+function useProvider() {
+  const room = useRoom();
+
+  const subscribe = useCallback(
+    (onStoreChange: () => void) => {
+      return room[kInternal].onProviderUpdate.subscribe(onStoreChange);
+    },
+    [room]
+  );
```

Comment:
This `useProvider` hook returns the currently attached provider in a room. In order to do so, we subscribe to any provider changes in the room and always return the most recently attached provider.